pkg/cache/memory.go

```diff
@@ -96,3 +96,17 @@ func (c *MemoryCache) Clear() error {
 	c.blobs = sync.Map{}
 	return nil
 }
+
+// BlobIDs returns all the blob IDs in the memory cache for testing
+func (c *MemoryCache) BlobIDs() ([]string, error) {
```

Comment:
This function never returns an error
```suggestion
func (c *MemoryCache) BlobIDs() []string {
```

Reply:
Thanks. Fixed in https://github.com/aquasecurity/trivy/pull/8403/commits/212b2a6448173647a8971a46a163add7cb32467d.